scripts/exregional_run_analysis.sh

```diff
@@ -110,6 +110,14 @@ case $MACHINE in
   APRUN="srun"
   ;;
 #
+"ORION")
+  ulimit -s unlimited
+  ulimit -a
+  export OMP_NUM_THREADS=1
+  export OMP_STACKSIZE=1024M
```

Comment:
Why is the stacksize requirement so different here than on Hera?

Reply:
This number works for NA 13km case on Orion. I didn't try other numbers.